server/zanata-war/src/test/java/org/zanata/adapter/GettextAdapterTest.java

```diff
@@ -105,5 +111,48 @@ public void testGettextReference() {
         assertThat(potEntryHeader.getReferences().get(0)).isEqualTo("reference");
     }
 
+    @Test
+    public void testTranslatedGettext() {
```

Comment:
I suggest the name `generateTranslatedGettext`, because we will probably need another test `parseTranslatedGettext`.